lib/logstash/config/mixin.rb

```diff
@@ -444,6 +445,12 @@ def validate_value(value, validator)
             end
 
             result = value.first
+          when :bytes
+            begin
+              result = Filesize.from(value.first).to_i
+            rescue ArgumentError
+              return false, "Unparseable filesize: #{value.first}"
+            end
```

Comment:
+1. Maybe offer an example "10KB" and then say "Possible units ...." and link to the docs?